File: trabajo 3/matriz_recorrer.java
public class matriz_recorrer {
    public static void main(String[] args) {
        int[][] arr = {
            {1, 2, 3},
            {1, 2, 3},
            {1, 2, 3}
        };

        // recorrido horizontal
        System.out.println("recorrido horizontal");
        for (int x = 0; x < 3; x++) {
            for (int y = 0; y < 3; y++) {
                System.out.print(arr[x][y] + " ");
            }
            System.out.println();
        }

        System.out.println("-------------");

        // recorrido vertical
        System.out.println("recorrido vertical");
        for (int x = 0; x < 3; x++) {
            for (int y = 0; y < 3; y++) {
                System.out.print(arr[y][x] + " ");
            }
            System.out.println();
        }
    }
}

File: trabajo 3/matriz_recorrer.cpp
#include <iostream>
using namespace std;
//recorrer arreglo

int arr[3][3] = {{1,2,3},{1,2,3},{1,2,3}}; //arreglo de matriz 3*3


int main(){
    //recorrido horizontal
    cout << "recorrido horizontal"<< endl;
    for(int x = 0; x < 3; x++){
        for(int y = 0; y < 3; y++){
            cout << arr[x][y] << " ";
        }
        cout << endl;
    }
    
    cout << "-------------"<< endl;
    
    //recorrido vertical
    cout << "recorrido vertical"<< endl;
    for(int x = 0; x < 3; x++){
        for(int y = 0; y < 3; y++){
            cout << arr[y][x] << " ";
        }
        cout << endl;
    }
    
return 0;
}